patches/src/main/kotlin/app/revanced/patches/spotify/misc/lyrics/Fingerprints.kt

```diff
@@ -0,0 +1,11 @@
+package app.revanced.patches.spotify.misc.lyrics
+
+import app.revanced.patcher.fingerprint
+
+/**
+ * This method builds and returns an HTTP client for a hardcoded host (declared in this method).
+ */
+internal val clientBuilderFingerprint = fingerprint {
```

Comment:
No reason to use multline
```suggestion
// Method builds and returns an HTTP client for a hardcoded host (declared in this method).
internal val clientBuilderFingerprint = fingerprint {
```